package.json

```diff
@@ -94,10 +94,11 @@
   "scripts": {
     "benchmark-rule": "node scripts/benchmark-rule.js",
     "flow": "flow",
-    "jest": "jest",
+    "jest": "jest --coverage",
     "lint:js": "eslint . --cache",
     "lint:md": "remark . --quiet --frail",
     "lint": "npm-run-all --parallel lint:*",
+    "preci": "npm-run-all --serial lint:md flow",
```

Comment:
Does this have something to do with the fact you are configuring ESLint and markdownlint in the `.codeclimate.yml` file and that we use remarklint rather than markdownlint?